website/docs/d/virtual_network.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
just noticed there isn't a sidebar entry for this Data Source - I've pushed a commit to fix this :)